data/examples/declaration/splice/untyped-splice-out.hs

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE TemplateHaskell #-}
+x = $(foo bar)
+
+x =
+  $( foo
+     bar
```

Comment:
This one seems weird to me. I think it should be something like this instead:

```haskell
x =
  $( foo
       bar
   )
```

What's inside the quote is simply a `HsExpr`, and I didn't want to modify that formatting logic in this PR. Am I using it wrong? Do I need to give more context about the indentation somehow?

Reply:
This can be fixed by using `sitcc` I guess.